### ISSUE TITLE
[Warning] Images should have a text alternative or presentational role (1)

### ISSUE THREAD

**Issue by rodowi** [mxabierto/participa#233]
/consulta/docs/manifesto-para-la-gobernanza-de-internet

[Warning] Images should have a text alternative or presentational role (1)
https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_02
```html
<img src="http://www.gob.mx/consulta-public/img/como-comentar.gif" class="how-to-annotate-img img-responsive">
```
